modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -50,7 +49,47 @@ Some actions are not yet supported:
 // We are waiting for SMT to release the feature/fix to mirror Debian repositories. When this has been done, this comment and the limitation above can be removed.
 
 
-This section describes how to manually prepare {ubuntu} clients for registration to your {susemgr} Server and synchronize the repositories.
+This section describes how to synchronize the repositories and prepare {ubuntu} clients for registration to your {susemgr} Server.
+
+
+== {ubuntu} Channel Management Tips
+
+This section provides tips on {ubuntu} channel management.
+
+* The base parent distribution {ubuntu} channel contains zero packages. No base media is provided by {suse}. The {ubuntu} media should be added as child channels of the {ubuntu} parent channel.
+* The {ubuntu} and tools channels are provided by SUSE Customer Center (SCC) using [command]``mgr-sync``.
+* When you have completed the initial synchronization process of any {ubuntu} channel it is recommended to clone the channel before working with it. This provides you with a backup of the original synchronization.
```

Comment:
Hmm, I know that this last point was copied from somewhere else, but I do not know if we should suggest cloning "here". I think it is more about a general section we get from the "Content Lifecycle Management" Feature.

Maybe we should put a reference to that chapter here instead of just say "clone everything" for backup.